omero/sysadmins/limitations.txt

```diff
@@ -26,6 +26,35 @@ correctly. OpenJDK and OracleJDK versions 6 and 7 are also supported.
 Windows OS issues
 -----------------
 
+Failure response on import for new databases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have import failures with an error such as::
+
+   java.lang.RuntimeException: Failure response on import!
+   Category: ::omero::grid::ImportRequest
+   Name: import-request-failure
+   Parameters: {stacktrace=java.lang.RuntimeException: omero.ValidationException
+       serverStackTrace = "ome.conditions.ValidationException: could not
+       insert: [ome.model.core.Pixels]; SQL [insert into pixels (creation_id,
+       external_id, group_id, owner_id, permissions, update_id,
+       dimensionOrder, methodology, physicalSizeXUnit, physicalSizeX,
+       physicalSizeYUnit, physicalSizeY, physicalSizeZUnit, physicalSizeZ,
+       pixelsType, relatedTo, sha1, significantBits, sizeC, sizeT, sizeX,
+       sizeY, sizeZ, timeIncrementUnit, timeIncrement, version, waveIncrement,
+       waveStart, image, image_index, id) values (?, ?, ?, ?, ?, ?, ?, ?, ?,
+       ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)];
+       nested exception is org.hibernate.exception.DataException: could not
+       insert: [ome.model.core.Pixels]
+
+when trying to import data to a new database (or possibly one just upgraded to
+5.1), it may be an issue with the language encoding during database
+initialization leading to corrupt unicode characters, such that any import
+containing these characters (e.g. µ) will fail with the above exception. If
+you are working with a fresh install, the easiest fix would be to drop the
+OMERO database and re-create it. Run ``SET CLIENT_ENCODING TO 'UTF8';`` in the
```

Comment:
Have we looked at setting PGCLIENTENCODING in the environment as an alternative; it might make the instructions simpler since you won't have to run psql interactively.